packager/packager.js

```diff
@@ -160,7 +160,13 @@ var server = runServer(options, function() {
   console.log('\nReact packager ready.\n');
 });
 
-webSocketProxy.attachToServer(server, '/debugger-proxy');
+var proxy = webSocketProxy.attachToServer(server, '/debugger-proxy');
+
+function isDebuggerConnected() {
+  // Debugger is connected if the app and at least one browser are connected
+  // to the websocket proxy.
+  return proxy.clientsCount() >= 2;
```

Comment:
If you are working on Android and iOS at the same time will this always be true?